lib/core/src/lib/clipboard/clipboard.directive.spec.ts

```diff
@@ -64,6 +64,15 @@ describe('ClipboardDirective', () => {
 
         expect(clipboardService.copyToClipboard).toHaveBeenCalled();
     });
+
+    it('should notify copy target value on mouse enter event', () => {
```

Comment:
Again, seems like incorrect title - it says about mouse enter but you have different event.

Reply:
title changed and added keydown instead of mouse enter.